src/test/java/Homework19.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.chrome.*;
+
+public class Homework19 extends BaseTest{
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+
+        BaseTest.openLoginUrl(url);
+        BaseTest.enterEmail(userEmail);
+        BaseTest.enterPassword(userPassword);
+        BaseTest.clickSubmitButton();
+        findAndOpenAPlaylist();
+        clickDeletePlaylistButton();
+        confirmPlaylistDeletedMessage();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**